packages/node_modules/samples/browser-call-with-screenshare/app.js

```diff
@@ -263,13 +262,172 @@ function joinMeeting(meeting) {
   });
 }
 
-// In order to simplify the state management needed to keep track of our button
-// handlers, we'll rely on the current meeting global object and only hook up event
-// handlers once.
+function handleRadio() {
```

Comment:
Let's use a more descriptive name on this:

```suggestion
function handleAuthTypeChange() {
```